accimagemodule.c

```diff
@@ -241,14 +241,28 @@ static PyTypeObject Image_Type = {
     0,                          /*tp_is_gc*/
 };
 
+#if PY_MAJOR_VERSION == 2
+#define PyUnicode_AsUTF8 PyString_AsString
+#define PyUnicode_Check PyString_Check
```

Comment:
I'm not sure this is going to work as strings in python 2 were bytes by default: `b'hi' == 'hi'`. This is why travis is failing at the moment. I'm not sure what we should do--drop python 2 support completely? (Although if torchvision is still supporting 2, so should accimage). If we need to support 2, perhaps we could ask for a bytesio object?

Reply:
I'll fix it, i think I can make the in-memory option be a kwarg-only option probably.